examples/js/loaders/GLTFLoader.js

```diff
@@ -3385,6 +3385,11 @@
 
 			}
 
+			if ( ! camera ) {
```

Comment:
According to the glTF spec, the projection can only be perspective or orthographic. How can `cameraDef.type` hold a different value without being invalid glTF?